raiden/network/proxies/token_network.py

```diff
@@ -106,7 +106,7 @@ def lock_or_raise(self, partner):
             lock = RLock()
             self.channel_operations_lock[partner] = lock
 
-        if not lock.acquire(blocking=False):
+        if not lock.acquire(blocking=True):
```

Comment:
This will never return `False`:

> If blocking is True and timeout is None (the default), then (so long as this semaphore was initialized with a size greater than 0) this will always return True.

So, I think we should remove this exception `ChannelBusyError` and change the code to:

```python
with lock:
  yield
```